server/api/controllers/inviteController.js

```diff
@@ -54,10 +61,33 @@ exports.createInvite = function(req, res) {
     console.log(JSON.stringify(err));
     if (err) res.status(500).send(err);
     invite['host_id'] = userId;
+    sendPushMessage(invite);
     res.json(invite);
   });
 };
 
+function sendPushMessage(invite) {
+  agSender(settings).then((client) => {
+      var message = {alert: 'invite'};
+      var alias = [invite.host_id]
+      invite.invitees.forEach(function(invitee){alias.push(invitee.id);});
```

Comment:
You know what, I forgot to add it to my hooks on my other computer.  It should be auto fixed now or soon though...